### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_coverlet.msbuild_Tab Navigation_Keyboard : Tab items 'Package Manager' , '.NET CLI' , 'PackageReference' etc in the tab lists are navigated using tab key instead of arrow key.

### ISSUE THREAD

**Issue by advay26** [NuGet/NuGetGallery#9068]
### Impact

It's more difficult to complete my work

### Describe the bug

**Source:** https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1468923

**Actual:**
Tab items 'Package Manager' , '.NET CLI' , 'PackageReference' etc in the tab lists are navigated using tab key instead of arrow key.
On using tab to a "tab" item the focus on next tab key moves to the next item rather then the data under it
Similar issue is observed for the tab list including 'Readme' , 'Dependencies' , 'Used by' and 'Version'.
Similar issue is observed in through out the application.

**Expected:**
Tab items 'Package Manager' , '.NET CLI' , 'PackageReference' etc in the tab lists should be navigated using arrow key instead of tab key and when focus is on a tab item the tab key focus should go inside the data under that tab item.
arrow keys should be used for navigation between tab items and tab key for the data under the tabs

**User Impact:**
Keyboard users will not be able to access the data under tabs immediately 

### Repro Steps

1.  Launch the application URL: https://www.nuget.org/packages/coverlet.msbuild/
2.  "coverlet.msbuild" page will get open.
3.  Tab to Tab navigation controls.
4.  Tab to Tab items 'Package Manager' , '.NET CLI' , 'PackageReference' etc in the tab lists and observe it.

### Expected Behavior

Tab items 'Package Manager' , '.NET CLI' , 'PackageReference' etc in the tab lists should be navigated using arrow key instead of tab key and when focus is on a tab item the tab key focus should go inside the data under that tab item.
arrow keys should be used for navigation between tab items and tab key for the data under the tabs
